### ISSUE TITLE
Beta page - better iframe sizing

### ISSUE THREAD

**Issue by lemeryfertitta** [lemeryfertitta/Climbdex#27]
Some widths and heights of the embedded instagram videos are a bit awkward to view, either on desktop, mobile, or both. Its possible that getting the post metadata could allow for better sizing, though my initial look into this wasn't very fruitful.

**Comment by gardaholm** [comment]
The Left / Right Icons are a bit off compared to the result page, at least on my Screens (mobie + desktop). Maybe this can be adresse too in with this issue.

**Comment by lemeryfertitta** [comment]
Good catch, I fixed those though its probably not worth a deploy, will just catch it in the next deploy.

The iframe sizing is a little trickier. I'm not even sure its possible to make it consistent since instagram has a whole bunch of different sizes that it gives you. The apps don't even bother with embedding and just send you to instagram, but I like the idea of embedding, I feel like it makes it a bit smoother to navigate.

**Comment by gardaholm** [comment]
I think it's ok as it is, it differs depending on the video sizes and if it was posted as video only or video in a slideshow. 

Overall its still easy to use and better to scroll throw the beta videos than using the app.
